src/Compression/ZipArchiveMember.class.st

```diff
@@ -205,33 +205,30 @@ ZipArchiveMember >> contentStreamFromEncoding: encodingName [
 	"Answer my contents as a text stream.
 	Interpret the raw bytes with given encodingName"
 
-	| s |
-	s := MultiByteBinaryOrTextStream on: (String new: self uncompressedSize).
-	s converter:  (TextConverter newForEncoding: encodingName).
-	self extractTo: s.
-	s reset.
-	^ s
-
+	| file |
+	file := MemoryFileSystemFile named: 'tempExtractFile'.
```

Comment:
The FileSystem-Memory package is not part of the basic system. It is loaded with Metacello but Zip support is already needed for loading packages using Monticello. That's why the build fails

Reply:
I have not tried, but would

```suggestion
    ^ ByteArray writeStreamDo: [ :stream | self extractTo: stream ] 
```

not work ?

Reply:
I think so too